types/kagekiri/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for kagekiri 1.0
+// Project: https://github.com/salesforce/kagekiri
+// Definitions by: Ted Conn <https://github.com/tedconn>
+//                 Nolan Lawson <https://github.com/nolanlawson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function querySelector(selector: string): HTMLElement | null;
+export function querySelectorAll(selector: string): NodeList;
```

Comment:
```suggestion
export function querySelector(selector: string, context?: Node): HTMLElement | null;
export function querySelectorAll(selector: string, context?: Node): NodeList;
```
what about allowing that context params?
https://github.com/salesforce/kagekiri#usage